nodejs/eks/examples/tests/migrate-nodegroups/nginx-ing-cntlr.ts

```diff
@@ -66,8 +66,8 @@ export function create(
     // Assemble the resources.
     // Per: https://itnext.io/kubernetes-ingress-controllers-how-to-choose-the-right-one-part-1-41d3554978d2
     const resources: input.core.v1.ResourceRequirements = {
-        requests: {memory: "1024Mi"},
-        limits: {memory: "2048Mi"},
+        requests: {memory: "1Gi"},
```

Comment:
Why is this change needed?

Reply:
See:
- https://github.com/pulumi/pulumi/issues/2967
- Slack [thread](https://pulumi.slack.com/archives/C8R654Y6S/p1563819398010100) from today